sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -965,6 +965,20 @@ public void isElementGreyedOut(By elementLocator) {
     }
   }
 
+  public void assertElementIsDisabled(By elementLocator) {
+    assertHelpers.assertWithPoll20Second(
+        () ->
+            Assert.assertTrue(
+                getAttributeFromWebElement(elementLocator, "class").contains("disabled")));
+  }
+
+  public void assertElementIsEnabled(By elementLocator) {
+    assertHelpers.assertWithPoll20Second(
+        () ->
+            Assert.assertFalse(
+                getAttributeFromWebElement(elementLocator, "class").contains("disabled")));
+  }
```

Comment:
we already have a method in place checking this: isElementEnabled
Is it possible to use it and maybe if is not working at that locator level, to find one where this one works?

If not possible, create a single method to check this, add it next to this existing one to make sure people will see both, and maybe name it like isElementEnabledAtAttributeLevel

Reply:
isElementEnabled returns true for everything but disabled input elements.
Resolved by creating new method, isElementEnabledAtAttributeLevel